code/__HELPERS/mobs.dm

```diff
@@ -616,3 +616,32 @@ GLOBAL_LIST_INIT(do_after_once_tracker, list())
 		chosen = pick(mob_spawn_meancritters)
 	var/mob/living/simple_animal/C = new chosen(spawn_location)
 	return C
+
+//determines the job of a mob, taking into account job transfers
+/proc/determine_role(mob/living/P)
+	var/datum/mind/M = P.mind
+	if(!M)
+		return
+	return M.playtime_role ? M.playtime_role : M.assigned_role	//returns current role
+
+//checks the security force on station and returns a list of numbers, of the form:
```

Comment:
Can you make this into a documentation that the auto documentation tool can use?
```
/**
  * documentation here
  * more here etc
*/

Reply:
Done, hope it's right like that.